### PR TITLE
fixes some rand() loops

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

i in rand(x,y) doesnt work

i in 1 to rand(x,y) does

## Why It's Good For The Game

bugggs

ghommie stole my gbp

## Changelog

:cl:
fix: fixed random dm modifiers & some life-like reagent reactions
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -571,7 +571,7 @@
 
 /datum/chemical_reaction/corgium/on_reaction(datum/reagents/holder, datum/equilibrium/reaction, created_volume)
 	var/location = get_turf(holder.my_atom)
-	for(var/i in rand(1, created_volume) to created_volume) // More lulz.
+	for(var/i in 1 to rand(1, created_volume) to created_volume) // More lulz.
```
Comment (SmArtKar) [inline code/modules/reagents/chemistry/recipes/others.dm:574]:
```suggestion
	for(var/i in 1 to rand(1, created_volume)) // More lulz.
```

Comment (MrMelbert) [reply code/modules/reagents/chemistry/recipes/others.dm:574]: The original code is correct tho

Comment (MrMelbert) [inline code/modules/reagents/chemistry/recipes/others.dm:574]: Well all of these are functioning, they're not broken, it's just a little obtuse

Comment (SmArtKar) [reply code/modules/reagents/chemistry/recipes/others.dm:574]: indeed, but this would be cleaner/more intuitive

Comment (carlarctg) [reply code/modules/reagents/chemistry/recipes/others.dm:574]: FUCK!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**